### PR TITLE
As peddy update

### PR DESCRIPTION
**Pull request by sickler-alex**
This is the final commit needed to close https://app.zenhub.com/workspaces/d3b-bfx-5cdd720eed0dce4209e0b8a5/issues/d3b-center/bixu-tracker/91
This updates the peddy latest docker to use the latest commit of peddy's master branch and creates a new docker for v0.4.7 which is currently the most recent version. These use python3 where the v0.4.2 version which is the current production version uses python2.

### CODE REVIEW

**Comment by migbro** [comment]
Oh, hmmm, I almost want to get rid of the `latest` tag rather than update it, otherwise we'll end up chasing their repo to make sure that is accurate. @zhangb1 , just a quick update, Alex have fixed the peddy Dockerfile and pinned the version. The latest public app has been updated and should point to the exact version now, and not latest. Thoughts on leaving the old `latest` docker alone, or just removing it to avoid confusion?

**Comment by dmiller15** [comment]
`latest` dockers that are their own thing are bad in my opinion. As the error from last night so eloquently pointed out, our versioning system in general could use a second look. But at a minimum, I think we should deprecate `latest` tags. Leave the old around until we can replace them; then delete them when they are no longer needed.

**Comment by sickler-alex** [comment]
I only kept the latest docker since that's what we already have.

**Comment by migbro** [comment]
Sure sure, it make sense and what you did was certainly rational. What we (probably I) did was a mistake using `latest`, so I think we are advocating for leaving the old `latest` docker and Dockerfile file as-is until we are ready to properly dispose of it...

**Comment by migbro** [comment]
I guess at worst, maybe adding a `LABEL` field to the old `latest` Dockerfile indicating that it is deprecated and no longer maintained will help bring clarity to anyone tempted to use it.

File: peddy/latest/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
 
-MAINTAINER Xiao Dang (dangx@email.chop.edu)
+LABEL maintainer="Alex Sickler (sicklera@email.chop.edu)"
 
-RUN apt-get update --fix-missing && apt upgrade -y &&  apt-get install -y wget python bzip2 git make sysstat g++ gcc zlib1g-dev zlib1g cmake build-essential libncurses5-dev libbz2-dev liblzma-dev libcurl4-gnutls-dev libcrypto++-dev libbamtools-dev libssl-dev python-pip python-dev build-essential \
-&& git clone https://github.com/brentp/peddy && cd /peddy && pip install numpy==1.14.5 && pip install cyvcf2==0.8.9 && pip install -r requirements.txt && pip install --editable . \
-&& apt-get clean build-essential libncurses5-dev libbz2-dev liblzma-dev libcurl4-gnutls-dev libcrypto++-dev libbamtools-dev libssl-dev && apt-get autoremove -y
+RUN apt-get update && apt upgrade -y \
+&& apt-get install -y python python3-pip git \
+&& git clone https://github.com/brentp/peddy && cd /peddy && pip3 install -r requirements.txt && cd /peddy && pip3 install -r requirements.txt && pip3 install . \
```
Comment (NHJohnson) [inline peddy/latest/Dockerfile]: Pardon my ignorance, but you `cd /peddy && pip3 install -r requirements.txt` twice?

**Review by NHJohnson — APPROVED** [review]
Looks good.

File: peddy/latest/Dockerfile (hunk 1 above)
Comment (sickler-alex) [reply peddy/latest/Dockerfile]: Good catch, I missed that. But, the latest version was reverted so it doesn't matter now.

**Review by migbro — APPROVED** [review]
Looks good, thanks! Definitely a good start to drop the `latest` and add to the description

**Pull request merged**